podcasts/EditFolderView.swift

```diff
@@ -87,3 +91,16 @@ struct EditFolderView: View {
         .navigationViewStyle(StackNavigationViewStyle())
     }
 }
+
+struct EditFolderView_Previews: PreviewProvider {
+    static var previews: some View {
+        Group {
+            ForEach (["iPhone 14 Pro", "iPhone 14 Pro Max", "iPhone 13 mini"], id: \.self) { simulator in
+                EditFolderView(model: FolderModel(), dismissAction: { _ in })
+                    .environmentObject(Theme(previewTheme: .light))
+                    .previewDevice(PreviewDevice(rawValue: simulator))
+                    .previewDisplayName(simulator)
+            }
+        }
+    }
+}
```

Comment:
We already have a modifier to allow testing in different devices:

```suggestion
struct EditFolderView_Previews: PreviewProvider {
    static var previews: some View {
        EditFolderView(model: FolderModel(), dismissAction: { _ in })
            .environmentObject(Theme(previewTheme: .light))
            .previewOnAllDevices()
    }
}
```

Reply:
@leandroalonso Thanks so much for pointing that out. I have committed your suggestion.